[DOCS] YCQL/YSQL snapshots fixes (#5432)
yugabyte/yugabyte-db: docs/content/latest/manage/backup-restore/snapshots-ycql.md

$ ./bin/ycqlsh
```

Create a keyspace, table, index, and insert some test data.

```sql
ycqlsh> CREATE KEYSPACE ydb;
ycqlsh> CREATE TABLE IF NOT EXISTS ydb.test_tb(user_id INT PRIMARY KEY, name TEXT) WITH transactions = {'enabled': true};
ycqlsh> CREATE INDEX test_tb_name ON ydb.test_tb(name);
ycqlsh> INSERT INTO ydb.test_tb(user_id,name) VALUES (5,'John Doe');
```

> You can verify that you have data in the database by running a simple SELECT statement.

Reviewer: ```suggestion
To verify that you have data in the database, run the following `SELECT` statement.
```

```sh
$ ./bin/yb-admin create_snapshot ydb test_tb
Started snapshot creation: a9442525-c7a2-42c8-8d2e-658060028f0e
```

> To see when your snapshot is ready, you can run the [`yb-admin list_snapshots`](../../../admin/yb-admin/#list-snapshots) command.

Reviewer: ```suggestion
To see when your snapshot is ready, run the [`yb-admin list_snapshots`](../../../admin/yb-admin/#list-snapshots) command.
```

Importing snapshot a9442525-c7a2-42c8-8d2e-658060028f0e (COMPLETE)
Table type: table
Table being imported: ydb.test_tb
Table type: index (attaching to the old table id cb612f9693fb40b6beeaa159078effd0)
Table being imported: ydb.test_tb_name
Successfully applied snapshot.
Object           	Old ID                           	New ID                          
Keyspace         	485a915f8f794308a6f39398040fada8 	6e407151f7ba41cf991f68dfdd5248b9
Table            	cb612f9693fb40b6beeaa159078effd0 	5550206e25d140698be031154a805823
Tablet 0         	6a2bf658a3ea47f0ba2515ce484096ad 	4da0ca52f96e4ed88f071196890550fd
Tablet 1         	5a8eb39732904f769c57033e1301c84c 	83b734a4e8d042a989a79a4340bc14e7
Keyspace         	485a915f8f794308a6f39398040fada8 	6e407151f7ba41cf991f68dfdd5248b9
Table            	6b538842e9f24f99b4b9ba2a995805fc 	2c59396c7e214a188dbdbcb3206b04d6
Tablet 0         	fa9feea93b0b410388e9bf383f938039 	456bf3575e6d41d2ba640386c1d9df26
Tablet 1         	1ac1047fb3354590968a6780fac89a67 	30fb27da04df46749129a42e9cf3289a
Waiting for table 2c59396c7e214a188dbdbcb3206b04d6...
Snapshot         	a9442525-c7a2-42c8-8d2e-658060028f0e 	27c331c0-4b5c-4027-85f9-75b7545641a7
```

> After you import the `metadata file`, you see some changes:

Reviewer: ```suggestion
After importing the `metadata file`, you see the following changes:
```

> 1. `Old ID` and `New ID` for table, tablets and snapshot.

Reviewer: ```suggestion
1. `Old ID` and `New ID` for table, tablets, and snapshot.
```

> 2. `table_id`, `tablet_id` and `snapshot_id` have changed, so you have a different paths from previously.

Reviewer: ```suggestion
2. `table_id`, `tablet_id` and `snapshot_id` have changed, therefore the paths are different.
```
